src/main/java/com/google/flourbot/api/CardResponse.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
this should be javadoc, so starts with /** and the comment should be immediately above the class definition.